ginga/misc/plugins/Cuts.py

```diff
@@ -120,6 +122,7 @@ def __init__(self, fv, fitsimage):
                                   show_cuts_legend=False, enable_slit=False)
         self.settings.load(onError='silent')
         self.colors = self.settings.get('colors', cut_colors)
+        self.show_legend = self.settings.get('show_cuts_legend', False)
```

Comment:
Yes, I realize that. However, I think it is necessary here because the plot area is shrunken 20% in `build_gui()` but the legend is not added/updated until `add_legend()` later. So, if the state of `'show_cuts_legend'` changed in between those two calls, things might get messy. Does this make sense?